paymentsheet/src/main/java/com/stripe/android/link/ui/wallet/WalletScreen.kt

```diff
@@ -722,7 +712,6 @@ internal const val WALLET_SCREEN_PAY_BUTTON = "wallet_screen_pay_button"
 internal const val WALLET_SCREEN_PAY_ANOTHER_WAY_BUTTON = "wallet_screen_pay_another_way_button"
 internal const val WALLET_SCREEN_RECOLLECTION_FORM_ERROR = "wallet_screen_recollection_form_error"
 internal const val WALLET_SCREEN_RECOLLECTION_FORM_FIELDS = "wallet_screen_recollection_form_fields"
-internal const val WALLET_SCREEN_BOX = "wallet_screen_box"
```

Comment:
assuming this wasn't required on e2e tests?

Reply:
Yep, not used.